lib/components/mobile/mobile.css

```diff
@@ -32,6 +32,13 @@
   display: inline-block;
 }
 
+@media (max-width: 800px) {
+  .otp .navbar .mobile-header-text {
+    font-size: 3vw;
```

Comment:
Really not sure about this one...  It seems to work but doesn't feel very stable.

Reply:
Will experiment with some javascript based solutions...

Reply:
Sometimes the header doesn't fit in the available space.

Reply:
This change ensures it always fits, but the text is sometimes a bit small

Reply:
Yes it is, and it kinda defeats the requirement for making the text legible. I think the bigger question is how do we handle text that is too long for the space available, regardless of the width. This is important for i18n. One workaround would be to show the title over two lines instead of one.

Reply:
@miles-grant-ibigroup Have you tried fitting the header text into two lines as we discussed last time?

Reply:
Do you remember what that fix was? I have a fix now using flex box but I'm not sure if that's the fix we found together.